api/src/formatters/csv.ts

```diff
@@ -29,9 +29,32 @@ export const csvParser: Parser = async (data: string) => {
   };
 };
 
+const csvInjectionProtector = (str: string) => {
+  const riskyChars = ["=", "+", "-", "@", ",", ";", "0", "1", "2", "3", "4", "5", "6", "7", "8", "9"];
```

Comment:
@bvenceslas from where did you get this function logic? Is it from https://github.com/destromas1/csv-injection-protector? If so, why you did not use the NPM package? Also, can you please add tests (we have a collection of tests in Traduora) to test different attempts to inject bad sequences and see how it's sanitized? Think you can use some payloads from https://github.com/payloadbox/csv-injection-payloads and also please review https://github.com/joanbono/GOCiS for some more test cases etc

Reply:
that package didn't work here, on other windows project using npm yes,
but not with yarn, it was crashing, so i thought about a logic and did.

Reply:
OK, but if you used code (copy / paste with some modifications) from that package, please add comments into files you copied like we are doing for example in https://github.com/ever-co/ever-gauzy/blob/develop/packages/core/src/core/core.module.ts:

```
// Copyright (c) 2021-2022 Ever Co. LTD
// Modified code from https://github.com/destromas1/csv-injection-protector 
// Originally MIT Licensed
// - see https://github.com/destromas1/csv-injection-protector/blob/master/LICENSE
// - original code `Copyright (c) 2019 Shahjada Talukdar`;
```

Thank you!